website/pages-parts/LandingPage/Testimonials.tsx

```diff
@@ -39,17 +46,33 @@ export function Testimonial({
             : styles['border-disable-top-right'],
         )}
       >
-        <h2>{who}</h2>
+        <img src={avatar} alt={who} className={styles['testimonial-avatar']} />
         <p>
-          <b>{company}</b>
+          <b>{who}</b>
+          <br />
+          {position} at <b>{company}</b>
           <br />
-          {position}
         </p>
+        <div className={styles['testimonial-links-wrapper']}>
+          <OvalLink to={linkGithub}>
+            <img
+              alt="github"
+              src="assets/github.svg"
+              className={styles['small-image-icon']}
+            />
+          </OvalLink>
+          <OvalLink to={linkLinkedin}>
+            <img
+              alt="github"
+              src="assets/linkedin.svg"
+              className={styles['small-image-icon']}
+            />
+          </OvalLink>
+        </div>
         <p
           className={classNames(styles.text, styles['testimonial-description'])}
-        >
-          {description}
-        </p>
+          dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
Instead of allowing arbitrary HTML here, make the `description` field a `ReactElement`.

Reply:
Good idea to get rid of dangerouslySetInnerHTML. Thanks!